libs/core/src/lib/components/wrapper-component.ts

```diff
@@ -251,9 +253,9 @@ export abstract class ReactWrapperComponent<TProps extends {}> implements AfterV
     };
   }
 
-  private _passAttributesAsProps() {
+  private _passAttributesAsProps() { 
     const hostAttributes = Array.from((this.elementRef.nativeElement as HTMLElement).attributes);
-
+    
```

Comment:
Please make sure to re-format your code with Prettier when committing to avoid inconsistencies like this.

Reply:
Acknowledged.